frame/society/src/migrations.rs

```diff
@@ -95,6 +104,14 @@ impl<
 	}
 }
 
+pub type VersionCheckedMigrateToV2<Runtime, Pallet, I, PastPayouts> = VersionedRuntimeUpgrade<
+	0,
+	2,
+	VersionUncheckedMigrateToV2<Runtime, I, PastPayouts>,
+	Pallet,
+	<Runtime as frame_system::Config>::DbWeight,
```

Comment:
I wonder if `Pallet` and `DbWeight` here can be combined into one `<T: Config>`? Ideally? You can always get `DbWeight` by `<T as frame_system::Confg>::DbWeight`. Not sure what the `Pallet` is used for.